openwisp_monitoring/device/static/monitoring/css/floorplan.css

```diff
@@ -0,0 +1,188 @@
+.menu-backdrop.active {
```

Comment:
Can we move the contents of this file into `device-map.css` to avoid one extra HTTP request for a few KB of CSS?

I'll simply separate it with a CSS comment.